Sources/OuterCore/CFG.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CFG.swift
+//  siltPackageDescription
+//
+//  Created by Robert Widmann on 2/20/18.
+//
+
+class CFG {
+  var rpo = [Int: Continuation]()
+
+  init(_ entry: Continuation, _ size: Int) {
+    _ = self.postOrderVisit(entry, size)
+  }
+
+  func postOrderVisit(_ n : Continuation, _ i_ : Int) -> Int {
+    var i = i_
+    for succ in n.successors {
+      if succ.forwardCFGIndex == -1 {
+        i = postOrderVisit(succ, i)
+      }
+    }
+    n.forwardCFGIndex = i - 1
```

Comment:
This, the dictionary, and the indices using the continuation as a cache all feel very wrong to me.  The CFG should centralize indexing and make RPO not cost a sorting pass thru an unordered collection